pom.xml

```diff
@@ -290,6 +291,12 @@
                 <version>${powermock.version}</version>
                 <scope>test</scope>
             </dependency>
+            <dependency>
+                <groupId>org.openjdk.jmh</groupId>
+                <artifactId>jmh-generator-annprocess</artifactId>
+                <version>${jmh.version}</version>
+                <scope>test</scope>
+            </dependency>
```

Comment:
Why I can't find the dependency on the DataCarrier pom.xml?

Reply:
I am also curious

Reply:
I mean you should, my local IDE may be set differently, but as you are adding this, we should make sure this is added in all necessary places.

Reply:
> I mean you should, my local IDE may be set differently, but as you are adding this, we should make sure this is added in all necessary places.

I have added this dependency to all necessary modules to ensure those benchmarks can be run locally. 

Also two general exclude rules are introduced to exclude all jmh generated classes.

```
**/generated/*_jmhType*.java,
**/generated/*_jmhTest.java
```

Reply:
```
**/generated/*_jmhType*.java,
**/generated/*_jmhTest.java
```

This is good.